atlas/templates/single.html

```diff
@@ -0,0 +1,29 @@
+<!-- wp:template-part {"slug":"header","tagName":"header"} /-->
+
+<!-- wp:group {"tagName":"main","style":{"spacing":{"margin":{"top":"var:preset|spacing|50"}}}} -->
+<main class="wp-block-group" style="margin-top:var(--wp--preset--spacing--50)">
+	<!-- wp:group {"layout":{"type":"constrained"}} -->
+	<div class="wp-block-group">
+		<!-- wp:post-title {"textAlign":"center","level":1,"style":{"spacing":{"margin":{"bottom":"var:preset|spacing|40"}}},"fontSize":"x-large"} /-->
+		<!-- wp:pattern {"slug":"atlas/post-meta"} /-->
+
+		<!-- wp:group {"align":"full","style":{"color":{"gradient":"linear-gradient(180deg,rgb(0,38,53) 0%,rgb(0,38,53) 50%,rgb(230,230,220) 50%,rgb(230,230,220) 100%)"},"spacing":{"margin":{"top":"0","bottom":"0px"}}},"layout":{"type":"constrained"}} -->
+		<div class="wp-block-group alignfull has-background" style="background:linear-gradient(180deg,rgb(0,38,53) 0%,rgb(0,38,53) 50%,rgb(230,230,220) 50%,rgb(230,230,220) 100%);margin-top:0;margin-bottom:0px"><!-- wp:post-featured-image {"overlayColor":"contrast","dimRatio":50,"align":"wide","style":{"spacing":{"margin":{"top":"var:preset|spacing|50"}}}} /--></div>
```

Comment:
I'm not a big fan of the light overlay. Could we make it the dark color? Should we use duotone here too?

Reply:
oh, I think the overlay is there from the parent theme or something! I don't like it either. I'm not too sure about the duotone, makes it all too dark